storage/gcloud/aio/storage/storage.py

```diff
@@ -172,6 +173,22 @@ async def _headers(self) -> Dict[str, str]:
             'Authorization': f'Bearer {token}',
         }
 
+    async def get_buckets(
+        self, project: str, timeout: int = DEFAULT_TIMEOUT,
+    ) -> Set[Bucket]:
+        url = ('https://storage.googleapis.com/'
```

Comment:
Please add a comment linking to the docs here, eg. just the following is fine:
```
# https://cloud.google.com/storage/docs/json_api/v1/buckets/list
```

Reply:
DONE!